packages/quick_actions/quick_actions_ios/ios/Classes/FLTQuickActionsPlugin.h

```diff
@@ -2,7 +2,13 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <Flutter/Flutter.h>
+@import Flutter;
 
 @interface FLTQuickActionsPlugin : NSObject <FlutterPlugin>
+
+/// Unavailable. Please use `initWithChannel:` instead.
```

Comment:
But that's "private", how would they be able to do it?  I think `registerWithRegistrar:` is the only way for developers to create an instance.

Reply:
Oh the unit tests will import the `Test` submodule that contains `initWithChannel:` private API. 

I think you made a good point that a public header shouldn't mention about this private API. I will remove it in next PR.